src/Psl/Collection/IndexAccessInterface.php

```diff
@@ -34,6 +34,17 @@ public function at(int|string $k): mixed;
      */
     public function contains(int|string $k): bool;
 
+    /**
+     * Alias of `contains`.
+     *
+     * @param Tk $k
+     *
+     * @see contains() method
+     *
+     * @psalm-mutation-free
+     */
+    public function containsKey(int|string $k): bool;
```

Comment:
This new method is not covered by the test-suite.

Reply:
Added. Thanks for the hint!

Reply:
Thanks. There are still some generated documentation and psalm issues left.
The infection task is fixed in a separate PR and is not related to this change.

Reply:
Almost there ... :)

```sh
make coding-standard-fix
make docs-generate
```

Reply:
Thanks!